ovos_tts_plugin_responsivevoice/__init__.py

```diff
@@ -49,7 +49,13 @@ def get_tts(self, sentence, wav_file):
         Returns:
             Tuple ((str) written file, None)
         """
-        self.engine.get_mp3(sentence, wav_file)
+        if lang and lang == self.lang:
```

Comment:
should be `if not lang or lang == self.lang`, otherwise the else will be called when `lang==None`

Reply:
Sorry, I realized this while adding the lang option to the other repos, already corrected